packages/core/__tests__/language-server/requests.test.ts

```diff
@@ -0,0 +1,328 @@
+
+          interface WrapperComponentSignature {
+            Blocks: {
+              default: [
+                {
+                  InnerComponent: WithBoundArgs<typeof EmberComponent, 'required'>;
+                  MaybeComponent?: ComponentLike<{ Args: { key: string } }>;
+                }
+              ];
+            };
+          }
+
+          export default class WrapperComponent extends Component<WrapperComponentSignature> {
+            logo = logo
+          }
+        `,
+      });
+
+      let server = project.startLanguageServer();
+
+      let formatting = ts.getDefaultFormatCodeSettings();
+      let preferences = {};
+      let edits = server.organizeImports(project.fileURI('index.ts'), formatting, preferences);
+
+      expect(edits).toEqual([
+        {
+          newText:
+            "import Component from '@ember/component';\nimport { ComponentLike, WithBoundArgs } from '@glint/template';\nimport './App.css';\nimport EmberComponent from './ember-component';\nimport logo from './logo.svg';\n",
+          range: {
+            start: { character: 0, line: 0 },
+            end: { character: 0, line: 1 },
+          },
+        },
+        {
+          newText: '',
+          range: {
+            start: { character: 0, line: 1 },
+            end: { character: 0, line: 2 },
+          },
+        },
+        {
+          newText: '',
+          range: {
+            start: { character: 0, line: 2 },
+            end: { character: 0, line: 3 },
+          },
+        },
+        {
+          newText: '',
+          range: {
+            start: { character: 0, line: 3 },
+            end: { character: 0, line: 4 },
+          },
+        },
+        {
+          newText: '',
+          range: {
+            start: { character: 0, line: 4 },
+            end: { character: 0, line: 5 },
+          },
+        },
+      ]);
+    });
+  });
+
+  test('many imports, already sorted', () => {
+    project.write({
+      'index.ts': stripIndent`
+        import Component from '@ember/component';
+        import { ComponentLike, WithBoundArgs } from '@glint/template';
+        import './App.css';
+        import EmberComponent from './ember-component';
+        import logo from './logo.svg';
+
+        interface WrapperComponentSignature {
+          Blocks: {
+            default: [
+              {
+                InnerComponent: WithBoundArgs<typeof EmberComponent, 'required'>;
+                MaybeComponent?: ComponentLike<{ Args: { key: string } }>;
+              }
+            ];
+          };
+        }
+
+        export default class WrapperComponent extends Component<WrapperComponentSignature> {
+          logo = logo
+        }
+      `,
+    });
+
+    let server = project.startLanguageServer();
+
+    let formatting = ts.getDefaultFormatCodeSettings();
+    let preferences = {};
+    let edits = server.organizeImports(project.fileURI('index.ts'), formatting, preferences);
+
+    expect(edits).toEqual([
```

Comment:
why are their edits if the imports are already sorted?

Reply:
based on this code: https://github.com/typed-ember/glint/pull/635/files#diff-cadffc59ac4fd3c9fd1cd7488980fc9cfea40cd5a93129f1021c54c810980e20R432

it looks like "that's just the upstream behavior" :shrug: